buildspec.yml

```
@@ -0,0 +1,63 @@
+    commands:
+      - echo Logging in to Docker Registry...
+      - echo "$DOCKER_REGISTRY_PASSWORD" | docker login -u "$DOCKER_REGISTRY_USERNAME" --password-stdin "$DOCKER_REGISTRY_URL"
+      - IMAGE_REPO_NAME="aws-node-todo-app-cicd"
+      - IMAGE_NAME="$DOCKER_REGISTRY_URL/$DOCKER_REGISTRY_USERNAME/$IMAGE_REPO_NAME"
+      - IMAGE_TAG=v1
+      
+  build:
+    commands:
+      - echo Build started on `date`
+      - echo Packaging application into a zip file...
+      - zip -r node-todo-app.zip * .[^.]* # Include all files and directories, including hidden files
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)